web/package.json

```diff
@@ -36,5 +36,10 @@
 	"type": "module",
 	"dependencies": {
 		"dotenv": "^16.4.5"
-	}
+	},
+	"overrides": { 
+    "vite": {
+      "rollup": "npm:@rollup/wasm-node"
+    }
+  }
```

Comment:
I don't understand why re-installing vite.
Didn't we agreed on using svelte @Lauwed ?

Reply:
Had a problem when installing the web dependencies on my machine (MacOS) . Found a similar issue and workaround here: https://github.com/vitejs/vite/issues/15167#issuecomment-1833274104

Reply:
I didn't take the time to read anything right now, but I know that SvelteKit is build on Vite.

Reply:
I can delete this change for this PR but I think other contributor on Mac could have this issue

Reply:
@pbrissaud can you still reproduce the problem with an up to date version of sveltekit ?

Reply:
@ferdodo Nope, I can create a new svelte-kit project on my Mac without any issues. 

And now, there are no more errors on this project when I delete the "overrides" part... I will delete it!

Reply:
@pbrissaud  perfect ! 🎉